tests/garage/tf/samplers/test_ray_batched_sampler_tf.py

```diff
@@ -14,6 +14,8 @@
 from garage.np.policies import ScriptedPolicy
 from garage.sampler import RaySampler, WorkerFactory
 from garage.tf.envs import TfEnv
+# pylint: disable=unused-import
+from tests.fixtures.sampler import ray_local_session_fixture  # noqa: F401
```

Comment:
if F401 and F811 are caused by pytest's requirements, just disable them for tests/ by editing:
* `setup.cfg` - `[flake8.per-file-ignores]`)
* `tests/garage/pylintrc`  - `[MESSAGES CONTROL.disable]`